kube-config-files/alphaville-series-rw-neo4j-red-service.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alphaville-series-rw-neo4j-red
+  labels:
+    app: alphaville-series-rw-neo4j-red
+    visualize: "true"
+spec:
+  replicas: 1
```

Comment:
Should we let only one instance in these services? If true please add a comment about it in each file.